Docs/Development.md

```diff
@@ -8,13 +8,13 @@ The easiest way to get started is to:
 
  1. Install Visual Studio.
  2. [Clone the repository](https://github.com/Microsoft/Git-Credential-Manager-for-Windows.git).
- 3. Open the solution file (GitCredentialManager.sln) using Visual Studio.
- 4. Right-click the solution node in Solution Explorer and choose 'Restore NuGet Packages'. This will download and setup all of the dependencies.
- 5. Right-click the 'Cli-CredentialHelper' project in Solution Explorer and select 'Properties'.
- 6. In the "Properties" window, select the 'Debug' tab from the left side.
- 7. In the "Properties: Debug" window, add the word "get" to the 'Command line arguments:' text box.
- 8. Close the "Properties" window.
- 5. Hit \<F5\>, or 'Debug' \>\> 'Start Debugging' from the top menu of Visual Studio.
+ 3. Open the solution file (`GitCredentialManager.sln`) using Visual Studio.
+ 4. Right-click the solution node in Solution Explorer and choose `Restore NuGet Packages`. This will download and setup all of the dependencies.
```

Comment:
Since these aren't referring to code elements, I believe the double- and single-quotes were correct as-is.